test/functional/inputs_test.rb

```diff
@@ -67,6 +68,88 @@
     end
   end
 
+  describe "when being passed inputs via the Runner API" do
+    let(:run_result) { run_runner_api_process(runner_options) }
+    let(:common_options) do
+      {
+        profile: "#{inputs_profiles_path}/via-runner",
+        reporter: ["json"],
+      }
+    end
+
+    # options:
+    #   profile: path to profile to run
+    #   All other opts passed to InSpec::Runner.new(...)
+    # then add.target is called
+    def run_runner_api_process(options)
+      # Remove profile from options. All other are passed to Runner.
+      profile = options.delete(:profile)
+
+      # Make a tmpfile
+      Tempfile.open(mode: 0700) do |script| # 0700 - rwx
```

Comment:
If the intention is to show what that mode 0700 does, I'd comment with `-rwx------`